aad/aad-b2c-with-spring-security/README.adoc

```diff
@@ -0,0 +1,207 @@
+
+=== Create user flows
+
+Follow the guide of https://docs.microsoft.com/azure/active-directory-b2c/tutorial-create-user-flows[AAD B2C user flows creation].
+
+=== Enable forgot password
+
+Follow the guide of https://docs.microsoft.com/en-us/azure/active-directory-b2c/add-password-reset-policy?pivots=b2c-user-flow#self-service-password-reset-recommended[AAD B2C enable forgot password].
+
+=== Enable profile edit
+
+Follow the guide of https://docs.microsoft.com/en-us/azure/active-directory-b2c/add-profile-editing-policy?pivots=b2c-user-flow[AAD B2C set up a profile edit flow].
+
+=== Enable password reset
+
+Follow the guide of https://docs.microsoft.com/en-us/azure/active-directory-b2c/add-password-reset-policy?pivots=b2c-user-flow#create-a-password-reset-user-flow[AAD B2C set up a password reset user flow]
+
+[[work-as-web-application]]
+== Work as web application
+
+[[work-application-simple-sample]]
+=== Web application simple sample
+
+Sample project: <<./web-application-1-simple-sample/README.adoc#chapter-link, web-application-1-simple-sample>>
+
+==== Fill necessary contents in sample project.
+
+1. Fill necessary contents in `application.yml`, like `${your-jwk-set-uri}`.
+
+2. Fill necessary contents in `home.html`, like `${your-tenant-name}`.
+
+==== Boot up the application
+
+Lunch the sample project, then open `http://localhost:8080` by browser.
+You are then redirected to the default _auto-generated_ login page. You can sign in or sign up with your account.
+
+Then, you can edit your profile with the button `Profile edit` or reset your password with `Password reset`.
+
+[[Web-application-with-scopes]]
+=== Web application with scopes
+
+Sample project: <<./web-application-2-with-scopes/README.adoc#chapter-link, web-application-2-with-scopes>>
+
+Compared with the previous sample, updated the `application.yml` and `SampleController.java`
+
+==== Configure scopes for application
+
+Configure scopes `Delegated.Premisson.Scope1` for application with https://docs.microsoft.com/en-us/azure/active-directory-b2c/add-web-api-application?tabs=app-reg-ga#configure-scopes[Configure-scopes].
+
+==== Fill necessary contents in sample project.
+
+1. Fill necessary contents in `application.yml`, like `${your-jwk-set-uri}`.
+
+2. Fill necessary contents in `home.html`, like `${your-tenant-name}`.
+
+==== Boot up the application
+
+Lunch the sample project, then open `http://localhost:8081` by browser.
+
+Compared with the previous sample, this sample adds scope for permission authentication.
+Please refer to <<Resource server authorize by scopes,Resource server authorize by scopes>>.
+
+[[Web-application-with-client-credentials]]
+=== Web application with client credentials
+
+Sample project: <<./web-application-3-with-client-credentials/README.adoc#chapter-link, web-application-3-with-client-credentials>>
+
+Compared with the previous sample, updated the `application.yml` and `WebSecurityConfiguration.java`
```

Comment:
Do you mean "Compared with the previous sample, this sample updated the `application.yml` and `WebSecurityConfiguration.java`" ?